spec/views/catalog/_edit_action.html.erb_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe "catalog/_edit_action.html.erb" do
+  let(:resource) do
+    GenericAsset.new do |g|
+      g.attributes = {
+        :id => 1,
+        :title => ["Title"],
+        :alternative => ["Test"]
+      }
+    end
+  end
+  let(:blacklight_config) {
```

Comment:
Do all these `let` blocks need to use real objects?  This test is coupled to quite a few real objects, which we've been trying to avoid where possible.